src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -75,11 +75,11 @@ type AttributeConfig = BaseAttributeConfig | BooleanAttributeConfig
 
 export interface Attributes {
   id: string
-  host: string | undefined
+  host?: string | undefined
   title?: string
   chartLibrary: ChartLibraryName
   width: number | string | null
-  height: number | string | null
+  height?: number | string | null
```

Comment:
Same here for optionals undefined is assumed (not sure if you need it as null here though...)

Reply:
@VLegakis `null` can actually be supported in some cases (in static custom dashboards) and have different behaviour (backwards compatibility), but thanks for asking! i will try narrow down those possibilities in the future, but i suspect that `null` in old static-custom-dashboards was resetting the height after resizing